### DISCUSSION TITLE
如何自定义网页标签的标题？

### DISCUSSION THREAD

**Discussion by fanmingfei** [umijs/umi#9376] [Q&A]
现在页面标签栏用的是路由配置的标题，有没有办法自定义标题？
比如现在这个页面的标题是 “如何自定义网页标签的标题？ · Discussion #9376 · umijs/umi”
我的标题也需要有一个当前页面的名字和当前网站名字的组合
我需要怎么解决这个问题？

**Comment by fz6m** [comment]
See https://github.com/nfl/react-helmet

**Reply by fanmingfei** [reply]
我在路由的组件里面使用helmet，他会先显示路由上配置的文字 再显示我在helmet里面的配置。一闪而过

**Reply by fz6m** [reply]
csr 项目都是这样的，第一次进入页面会显示 `index.html` 的标题，之后在应用内 push 跳转不会有这个问题。

如需修改 `index.html` 默认标题，请使用 [title](https://umijs.org/docs/api/config#title)

**Reply by fanmingfei** [reply]
进入页面的时候显示的是路由的title不是index.html的title

**Reply by fz6m** [reply]
给一个复现看看

**Reply by fanmingfei** [reply]
有没有现成的 codesandbox模板。我看上面是3.x版本的
